lexus_rx_450h_2019/SubsystemControllerParams.yaml

```diff
@@ -47,20 +47,19 @@ guidance:
         - /guidance/plugins/pure_pursuit_wrapper
         - /guidance/plugins/inlanecruising_plugin
         - /guidance/plugins/cooperative_lanechange
-      auto_activated_plugins:
+      auto_activated_plugins: # TODO: Current logic requires that at most one control be activated at a time. This will be addressed under CAR-6064. https://usdot-carma.atlassian.net/browse/CAR-6064
```

Comment:
Not related to this specific file, but looks like we need to add the changes to the chrysler_pacifica_ehybrid_s_2019 config as well

Reply:
Good catch, thank you!